controllers/ControllerAuth.php

```diff
@@ -31,7 +33,7 @@ public function login() {
 
 	public function logout(){
 		session_destroy();
-		header("Location: http://localhost:8000/Home");
+		header("Location: http://".$_SERVER['SERVER_NAME']."/Home");
```

Comment:
Codacy found an issue: [The use of function header() is discouraged](https://app.codacy.com/manual/PatOpen/OC_Blog/pullRequest?prid=5982252)